modules/install-cleanup/install.sh

```diff
@@ -48,10 +48,6 @@ function aggregate_patched_modules {
   fi
 
   rm -rf /tmp/standalone
-
-  if [ -n "$sys_pkgs" ]; then
```

Comment:
@bstansberry After testing, this does not appear to be still needed. I'm not 100% clear on what changed to allow it to work though...